src/sidebar.js

```diff
@@ -209,6 +202,36 @@ class Sidebar extends React.Component {
 
     let rootProps = {
       style: styles.root,
+    };
+
+    rootProps.style.top = this.props.topSidebar;
+
+    if (this.props.pullRight) {
+      if (this.props.enableShadow) {
+        sidebarStyle.boxShadow = '-2px 2px 4px rgba(0, 0, 0, 0.15)';
```

Comment:
Please do not modify the `sidebarStyle` style property directly but only create new ones using the `update` method (as is done 2 lines below). Probably better to merge the two calls to `update` in one:

```javascript
const sidebarStyleToMerge = {
  right: 0,
  transform: 'translateX(100%)',
  WebkitTransform: 'translateX(100%)',
}

if (this.props.shadow) {
  sidebarStyleToMerge.boxShadow = '-2px 2px 4px rgba(0, 0, 0, 0.15)';
}

sidebarStyle = update(sidebarStyle, { $merge: sidebarStyleToMerge });
```